rust/forevervm-sdk/src/api/api_types.rs

```diff
@@ -60,11 +60,13 @@ impl Instruction {
 }
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+#[serde(untagged, rename_all = "snake_case")]
```

Comment:
@paulgb i would really prefer to use a tagged union here but the `untagged` is required for it to be backwards compatible with the existing JSON (where `value` is just an optional string rather than a nested object)